Project-3/Peihong_Xie_Project_3_50375781.md

```diff
@@ -0,0 +1,381 @@
+
+(b) The object property **"determined by"** should be transitive, though it is not listed as so. That is, if A is determined by B, and B is determined by C, then A should be determined by C. For example, a small-size ecological cycle is determined by a biological host, say, a human being, that is, as a system at another level of granularity, is determined by his brain. In this case, we can say that the ecological cycle is determined by the human being's brain because its proper functioning leads the human to make efforts in maintaining that cycle. 
+
+(c) The object property **"biotically interacts with"** should be symmetric, though it is not listed as so. If an organism A biotically interacts with another organism B, then it is clear that B is also biotically interacts with A. 
+
+Similarly, **"participates in a biotic-biotic interaction with", "ecologically co-occurs with", "partially overlap" and so on** should also be symmetric.
+
+(d) The object property **"has skeleton"** should be functional, though it is not listed as so. According to its definition, "has skeleton" is a relation between a segment or subdivision of an organism and the maximal subdivision of material entities that provides structural support for that segment or subdivision. Given that the maximal subdivision of material entities for any organism has only one, this object property is thereby functional.
+
+(e) The object property **"overlaps"** should be both symmetric and reflexive, though it is not listed as so. First, if A overlaps B, then A and B has some common part. So B and A has also some common part, and thus B overlaps A. That is, overlapping is symmetric. Second, an entity can overlap itself because it clearly has the same parts with itself. Therefore, overlapping is also reflexive.
+
+Similarly, **"spatially coextensive with" and "simultaneous with"** is also symmetric and reflexive.
+
+**[2]** In BFO and RO identify at least one object property for each of a-e that _should not have the listed property, but which does_; argue for your case, using examples. 
+```
+  (a)  Irreflexive
+  (b)  Transitive 
+  (c)  Asymmetric
+  (d)  Functional 
+  (e)  Inverse Functional
+```
+
+**Answer:**
+
+(a) Although **"has substance added"** is listed as irreflexive, but it should not be so. That is, in some cases, a physical entity can have itself as substance added to it. For example, a portion of water can be added to itself at some time. Similarly, **"has substance removed"** should not be irreflexive, either because we can always remove the same amount of water from itself.
+
+(b) Although **"aligned with"** is listed as transitive, but it should not be so. Suppose that we have three different quadrilateral objects A, B, and C. A is aligned with B because of their alighed long sides, and B is aligned with C because of their aligned short sides, but A is not thus aligned with C because neither their long nor short sides are not aligned with each other.
```

Comment:
Aren't they aligned with respect to spatial orientation? The definition is (from PATO; I couldn't find the RO definition): An alignment quality inhering in a bearer by virtue of the bearer's being in a proper spatial positioning with respect to an additional entity.